core/tests/puppeteer-acceptance-tests/specs/exploration-editor/create-exploration-and-change-basic-settings.spec.ts

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 /**
- * @fileoverview Acceptance Test for Exploration Creator and Exploration Manager
+ * @fileoverview Acceptance Test for settings tab in exploration editor.
```

Comment:
This description is not clear enough. I suggest changing it to "creating and modifying an exploration on/via the settings tab"

Reply:
Done